circuits/cpp/src/aztec3/circuits/kernel/private/.test.cpp

```diff
@@ -66,69 +66,53 @@ TEST_F(private_kernel_tests, circuit_basic)
     EXPECT_TRUE(private_kernel_builder.check_circuit());
 }
 
+// TODO(1998): Lack of support for msgpack deserialization for Circuitresult type for this test
+// to be able to call private_kernel__sim_init.
 /**
  * @brief Some private circuit simulation checked against its results via cbinds
  */
+/*
 TEST_F(private_kernel_tests, circuit_cbinds)
```

Comment:
If you wish to disable a test, you can simply do:
`TEST_F(private_kernel_tests, DISABLED_circuit_cbinds)`

Reply:
So you won't need to comment it out